Sources/ContainerizationEXT4/EXT4+Formatter.swift

```diff
@@ -1266,7 +1266,7 @@ extension EXT4 {
             case invalidName(_ name: String)
             case noSpaceForTrailingDEntry
             case insufficientSpaceForGroupDescriptorBlocks
-            case cannotCreateHardlinkstoDirTarget(_ path: FilePath)
+            case cannotCreateHardlinksToDirTarget(_ path: FilePath)
```

Comment:
This would be a breaking change (although is correct). That's fine given the projects current state, but commenting so other maintainers know we'll need to tag a 0.2.0.